sdk/java/src/main/java/com/chain/http/Client.java

```diff
@@ -5,11 +5,19 @@
 
 import java.io.*;
 import java.lang.reflect.Type;
+import java.math.BigInteger;
 import java.net.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.security.GeneralSecurityException;
+import java.security.KeyFactory;
```

Comment:
We'd have to do it at every package level, i.e. `java.security.*`, `java.security.cert.*`, etc. and if these packages use matching class names there will be conflicts. I'd rather leave it as is.